modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingPlusFeaturesPage.kt

```diff
@@ -134,11 +142,13 @@ fun OnboardingPlusFeaturesPage(
                 Spacer(Modifier.weight(1f))
                 Spacer(Modifier.height(36.dp))
 
-                PlusRowButton(
-                    text = stringResource(LR.string.onboarding_upgrade_unlock_all_features),
-                    onClick = onUpgradePressed,
-                    modifier = Modifier.padding(horizontal = 24.dp),
-                )
+                if (canUpgrade) {
```

Comment:
This `canUpgrade` variable should never be false because it will only be false if there aren't any subscriptions available. I have it just as a bit of a safety check so that if for some reason a user doesn't see any subscriptions instead of asking them to select from a list of 0 subscriptions, the only option we give them is to tap on "Not Now". That's still not a great experience, but it seems a lot better than the alternative and, like I mentioned, it shouldn't really ever come up.